msi/GitAdditionalOptionsDlg.wxs

```diff
@@ -6,7 +6,7 @@
         <Control Type="Text" Id="EnableFsCacheText" Width="275" Height="30" X="38" Y="72" TabSkip="no" Text="!(loc.GitAdditionalOptionsDlg_EnableFSCacheText)" />
         <Control Type="CheckBox" Id="EnableFsCache" Width="324" Height="15" X="20" Y="53" Integer="yes" CheckBoxValue="1" Property="UI_ENABLEFSCACHE" Text="!(loc.GitAdditionalOptionsDlg_EnableFsCacheCheckBox)" />
         <Control Type="Text" Id="EnableGcmText" Width="275" Height="40" X="38" Y="135" TabSkip="no" Text="!(loc.GitAdditionalOptionsDlg_EnableGcmText)" />
-        <Control Type="CheckBox" Id="EnableGCM" Width="324" Height="15" X="20" Y="116" CheckBoxValue="1" Integer="yes" Property="ENABLEGCM" Text="Enable Git Credential Manager for Windows">
+        <Control Type="CheckBox" Id="EnableGCM" Width="324" Height="15" X="20" Y="116" CheckBoxValue="1" Integer="yes" Property="ENABLEGCM" Text="!(loc.GitAdditionalOptionsDlg_EnableGcmCheckBox)">
```

Comment:
Actually, I changed my mind. I think I'm going to make an exception here and leave this. I've cleaned this up a lot in an upcoming update, and I think this commit will tell a great story about how "configuring" Enable Git Credentials Manager for Windows works, as a whole. Also, the condition on the checkbox (see the hunk right below this comment chain) requires a change being made in this commit, also. Without that change, the MSI wouldn't build until we came all the way down to this commit.

So, the initial UI commit shows the basic "shape" of the UI. The initial Remember Property commit shows the bare minimum to implement the property. This commit fleshes everything out, with an extremely explanatory commit message about the details. (I'm going to remove a lot of the comments in the code--they're too long and distracting.)